src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs

```diff
@@ -99,30 +69,39 @@ public void TearDown()
             channelProvider?.Dispose();
         }
 
-        protected IncomingMessage WaitForMessage()
+        protected bool TryWaitForMessageReceipt()
```

Comment:
I always think of the Try pattern as requiring an out parameter, though I suppose it strictly speaking doesn't need one. Is there an example in the BCL that doesn't?

So, in this scenario, to fit the pattern, the non-try method signature would be `void WaitForMessageReceipt()` and since the return type here is `void` the Try version would be `out void` which is meaningless, so you get `bool TryWaitForMessageReceipt()` instead.

Ok, I'm good with this then.